common-util/Login/LoginV2.jsx

```diff
@@ -79,9 +79,7 @@ export const LoginV2 = ({
 
   return (
     <LoginContainer>
-      <Web3NetworkSwitch />
-      &nbsp;&nbsp;
-      <Web3Button balance="show" avatar="hide" />
+      <Web3Button balance="hide" avatar="hide" />
```

Comment:
This is too big and provides little additional value on this site above the button that's already in web3modal